src/domain/event/eventForm/EventForm.tsx

```diff
@@ -518,6 +523,16 @@ const EventFormWrapper: React.FC<EventFormWrapperProps> = (props) => {
   const { event } = props;
   const { user, externalUser } = useUser();
 
+  const { organizationAncestors } = useOrganizationAncestors(
+    getValue(event?.publisher, '')
+  );
+
+  const isAdminUser = isAdminUserInOrganization({
+    id: getValue(event?.publisher, ''),
+    organizationAncestors,
+    user,
+  });
```

Comment:
`organizationAncestors` and `isAdminUser` could be moved to EventForm component EventFormWrapper doesn't use them. Or is there something I'm missing?

Reply:
I think I was planning to do something with them at first but then didn't need to 😁

Reply:
https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/251/commits/13d1304823525678db1873e17506fc984531e410